CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# release configuration
```

Comment:
We've got an existing [CODEOWNERS file](https://github.com/hashicorp/consul-terraform-sync/blob/main/.github/CODEOWNERS) under the `.github` directory, could this be added to that file or does it need to be at the root level?

Reply:
My apologies, I overlooked this subdir in a couple of repos - I've updated this to use the existing CODEOWNERS and will proceed to verify I haven't repeated the mistake in other consul ecosystem repos.